addons/website_form/static/src/js/website_form_editor.js

```diff
@@ -166,6 +167,61 @@ odoo.define('website_form_editor', function (require) {
             });
         },
 
+        /**
+         * Prompt the user for a default value for this field.
+         *
+         * It creates a modal with a clone of the target field.
+         */
+        ask_default_value: function () {
+            this.trigger_up("clean_for_save");
```

Comment:
I am not sure why you would need that ?

Reply:
Because some fields render differently depending on if you are editing them or they are clean, and you need a clean input to be cloned into the modal.